assets/js/modules/analytics/components/setup/SetupFormGA4Transitional.js

```diff
@@ -34,7 +34,10 @@ import {
 	PROPERTY_TYPE_GA4,
 	ACCOUNT_CREATE,
 } from '../../datastore/constants';
-import GA4PropertySelect from '../../../analytics-4/components/common/PropertySelect';
+import {
+	PropertySelect as GA4PropertySelect,
+	WebDataStreamSelect as GA4WebDataStreamSelect,
```

Comment:
We don't need to alias `WebDataStreamSelect` component since it's already GA4-specific :)

Reply:
Yes, I know, but I added it just for consistency with the property select dropdown for the Analytics-4 module. I think in this case it is clearly that those components are for GA4. If you feel strongly about it, I can remove that alias, not a big deal for me. Just let me know 😄